src/Microsoft.Maintainability.Analyzers/UnitTests/Microsoft.Maintainability.Analyzers.UnitTests.csproj

```diff
@@ -162,6 +162,9 @@
     <Compile Include="AvoidUnusedPrivateFieldsTests.cs" />
     <Compile Include="AvoidUnusedPrivateFieldsTests.Fixer.cs" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
I didn't see these Include in other csproj files except for Microsoft.Maintainability.Analyzers.UnitTests.csproj.